client/app/src/main/java/com/example/mrides/Notification/InboxAdapter.java

```diff
@@ -15,11 +16,14 @@
 public class InboxAdapter extends RecyclerView.Adapter<InboxAdapter.ViewHolder>{
 
     private Map<String,String> responseBody;
-    private static String inviteMessage;
+    private Context inboxContext;
 
-    public InboxAdapter(Map<String,String> responseBody, String inviteMessage){
+    public InboxAdapter(Context inboxContext){
```

Comment:
There should be a space between the closing parenthesis and opening curly bracket.